egs/tedlium3/ASR/transducer_stateless/beam_search.py

```diff
@@ -0,0 +1,352 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
```

Comment:
Can you use the latest `beam_search.py` from the master? It is a little faster

Reply:
OK, will use it.